src/components/Router.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import RoutePath from './RoutePath';
+import { RoutePathProps } from '../types/component-types';
+
+const AppRouter: React.FC<RoutePathProps> = ({ key }) => {
```

Comment:
What does this do? It's not connected to the app at all. You can delete this file.

Also components folder is usually reserved for components. A router file is not usually in a components folder.